poap-xyz/poap-gallery
# ✨ Add: Lazyloading images with POAP placeholder

## Summary

POAP images are not being optimized when uploading/rendering and they usually take time to load on client side. With this improvement images are now loaded with a new component [LazyImage](https://github.com/poap-xyz/poap-gallery/tree/feature/PCD-602-lazy-load-images/src/components/LazyImage) that shows a SVG placeholder until the real image finishes loading.

## Type of change


## Screenshots

![Lazyloading images with POAP placeholder](https://user-images.githubusercontent.com/11730998/140417329-7925de6a-7965-4d16-93ed-45070a638954.jpg)
*Gallery View*
 
![image](https://user-images.githubusercontent.com/11730998/140417420-043fa981-13a5-4e77-a031-6d4691a5ff84.png)
*Activity View*

## Testing

* Check that the placeholder is showing correctly before image has finished loading.
* Check that POAPs images are showing correctly along the whole site (mobile & desktop).

## Ticket

[PCD-602](https://poap-devs.atlassian.net/browse/PCD-602)

## Dependencies

None

## Checklist:
